### PR TITLE
Homepage,login,and register#1

### PR DESCRIPTION
**Pull request by kenichi127**
Description / purpose (check point)
Go to each page (login, registration, search results) from the homepage.

Fixes issue: 
https://docs.google.com/spreadsheets/d/1BHy8K_EvpG4fBLjujxShiZWe2-tNs0Jj/edit#gid=905621989
task-number:1,4,6,and9

How to test this PR
Login
1Click Login on the Home
2 Enter information
3 Click login

Register
1 Click on register of homepage
2 Enter information
3 Click register
 
Search results
1Select the date of home page
2Click Search

Commands
php artisan migrate
Screenshot
Homepage
image
Login
image
Register
image
After Login and Register
image
After Search
image

### CODE REVIEW

File: app/Http/Controllers/Auth/RegisterController.php [hunk 1]
```diff
@@ -1,13 +1,10 @@
 <?php
-
```
Comment (AubreyV) [inline app/Http/Controllers/Auth/RegisterController.php]: Please do not remove the space here; the space is intended so that it will be easier to read the code.

File: app/Http/Controllers/Auth/RegisterController.php [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
 namespace App\Http\Controllers\Auth;
-
 use App\User;
 use App\Http\Controllers\Controller;
 use Illuminate\Support\Facades\Hash;
 use Illuminate\Support\Facades\Validator;
 use Illuminate\Foundation\Auth\RegistersUsers;
-
```
Comment (AubreyV) [inline app/Http/Controllers/Auth/RegisterController.php]: do not remove also in here

File: app/Http/Controllers/Auth/RegisterController.php [hunk 3]
```diff
@@ -20,16 +17,13 @@ class RegisterController extends Controller
     | provide this functionality without requiring any additional code.
     |
     */
-
```
Comment (AubreyV) [inline app/Http/Controllers/Auth/RegisterController.php]: and here

File: app/Http/Controllers/Auth/RegisterController.php [hunk 4]
```diff
@@ -20,16 +17,13 @@ class RegisterController extends Controller
...
     use RegistersUsers;
-
```
Comment (AubreyV) [inline app/Http/Controllers/Auth/RegisterController.php]: this part also

File: app/Http/Controllers/Auth/RegisterController.php [hunk 5]
```diff
@@ -20,16 +17,13 @@ class RegisterController extends Controller
...
     /**
      * Where to redirect users after registration.
      *
      * @var string
      */
     protected $redirectTo = '/home';
-
```
Comment (AubreyV) [inline app/Http/Controllers/Auth/RegisterController.php:32]: here too

File: app/Http/Controllers/Auth/RegisterController.php [hunk 6]
```diff
@@ -39,7 +33,6 @@ public function __construct()
     {
         $this->middleware('guest');
     }
-
```
Comment (AubreyV) [inline app/Http/Controllers/Auth/RegisterController.php:42]: here too

File: app/Http/Controllers/HomeController.php [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('home');
+    }
+    public function post(Request $request)
+    {
+        // Auth::user()->blogs()->create()
+        $blog = new Blog;
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: Why do you have blog here?

File: app/Http/Controllers/UserController.php [hunk 8]
```diff
@@ -0,0 +1,27 @@
+<?php
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
+use App\Blog;
```
Comment (AubreyV) [inline app/Http/Controllers/UserController.php]: Why do you have blog here? You do not have Blog.php Model and you are not using Blog for this project.

File: app/Http/Controllers/UserController.php [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        $auth_id = Auth::user()->id;
+        $users = User::where('id', '!=', $auth_id)->get();
+        // dd(auth()->user()->following()->find());
+        return view('users', compact('users'));
+    }
+    public function user_home($id)
+    {
+        $user = User::find($id);
+        $blogs = $user->blogs()->get();
```
Comment (AubreyV) [inline app/Http/Controllers/UserController.php]: blog is here again and you are not using blog in this project

File: app/Http/Controllers/UserController.php [hunk 10]
```diff
@@ -17,11 +16,5 @@ public function users()
         // dd(auth()->user()->following()->find());
```
Comment (AubreyV) [inline app/Http/Controllers/UserController.php]: Please remove this dd().

**Comment by hayato0309** [comment]
1. After clicking "search" button in /home page, it becomes 404.
https://gyazo.com/87291115a69c9dad6a713228a3d5f076

File: app/Mail/PasswordResetNotification.php [hunk 11]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (AubreyV) [inline app/Mail/PasswordResetNotification.php]: Why is password reset already in this PR?
It should be in a different PR because it's another task.

**Comment by hayato0309** [comment]
1. This becomes one line.
https://gyazo.com/0f4d202e52e7f6fe688596d892edd9e4
Based on your screen shot, it should be in 2 lines.

**Review by AubreyV — APPROVED** [review]

**Comment by hayato0309** [comment]
1. Please place the "Login" button in one line.
